hustle/main/templates/main/other_profile.html

```diff
@@ -8,13 +8,57 @@
 
 <!--Register-->
 <div class="row my-4">
+    {% if view_user.customer_data %}
     <div class="col-3">
         <div class="card text-center mx-2">
             <div class="card-header bg-brown text-white">
-                <h2>Recent Activity</h2>
+                <h2>Worker Surveys</h2>
+            </div>
+            <div class="card-body scroll">
+                {% for survey in surveys %}
+                <div class="card mx-4 mb-4">
+                    <div class="card-header text-center">
+                        <h2 class="text-center">Job #{{ survey.job.id }}</h2>
+                        <strong>Type: {{survey.job.type }}</strong>
+                    </div>
+                    <div class="card-body">
+                        <strong>Duration Notes: </strong>
+                        <p>{{ survey.duration_notes }}</p>
+                        <strong>Difficulty Notes: </strong>
+                        <p>{{ survey.complexity_notes }}</p>
+                        <strong>Other Notes: </strong>
+                        <p>{{ survey.notes }}</p>
+                        <strong>By: {{ survey.job.claimed_user.first_name }} {{ survey.job.claimed_user.last_name }}</strong>
+                        <p>Date: {{ survey.create_date }}</p>
+                    </div>
+                </div>
+                {% endfor %}
+            </div>
+        </div>
+    </div>
+    {% elif view_user.worker_data %}
+    <div class="col-3">
+        <div class="card text-center mx-2">
+            <div class="card-header bg-brown text-white mb-4">
+                <h2>Customer Reviews</h2>
+            </div>
+            <div class="card-body scroll">
+                {% for review in reviews %}
+                <div class="card mx-4 mb-4">
+                    <div class="card-header text-center">
+                        <h2 class="text-center">Rating: {{ review.rating }}/5 </h2>
+                    </div>
+                    <div class="card-body">
+                        <p>{{ review.comments }}</p>
+                        <strong>By: {{ review.reviewer.first_name }} {{ reivew.reviewer.last_name }}</strong>
```

Comment:
Misspelled `review` as `reivew`
```suggestion
                        <strong>By: {{ review.reviewer.first_name }} {{ review.reviewer.last_name }}</strong>
```